### DISCUSSION TITLE
stored the result of a formula as a normal data in the database

### DISCUSSION THREAD

**Discussion by oswaldB** [nocodb/nocodb#546] [Q&A]
Hello,

I don't know if it is possible so I ask the question:
Is it possible to store the return of a formula as a data in the database?

For example I have column A (int) and column B (int) and I want to save the sum of them in column C, so I use the formula. In NocoDb I can see the result (calculation in front I imagine) but when I want to have access to this data outside nocoDb i cannot find it. It is not store in the database.

Thank you very much for your answers.

**Comment by o1lab** [comment]
Storing that in database would mean its a column :)

**Comment by oswaldB** [comment]
It would be awesome to save the formula answer in a column then ;)
